src/current/v23.1/alter-table.md

```diff
@@ -911,6 +911,50 @@ $ cockroach demo bank
 (1 row)
 ~~~
 
+##### Move a column from one column family to another
+
+To move a column from one column family to another column family, create a temporary, [non-visible](create-table.html#not-visible-property) [stored computed column](computed-columns.html) in the target column family, then rename the columns. Once this succeeds, you can drop the original, now renamed column.
+
+For example, to move the `new_name` column from `f2` to `f1`:
+
+1. Create a temporary computed column in the target column family of the same data type as the column you want to move:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER TABLE bank
+        ADD COLUMN newer_name STRING
+        FAMILY f1 NOT VISIBLE AS (new_name) STORED;
+    ~~~
+
+    This causes `newer_name` to have the same values as `new_name`.
+
+1. Rename the columns:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER TABLE bank
+        ALTER COLUMN newer_name DROP STORED,
+        ALTER COLUMN newer_name SET VISIBLE,
+        RENAME COLUMN new_name TO old_name,
+        RENAME COLUMN newer_name TO new_name,
+        ALTER COLUMN old_name SET NOT VISIBLE;
+    ~~~
+
+1. Drop the old column:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET sql_safe_updates = false;
+    ALTER TABLE bank DROP COLUMN old_name;
+    SET sql_safe_updates = true;
+    ~~~
+
+    {{site.data.alerts.callout_info}}
+    You must set the `sql_safe_updates` [session variable](set-vars.html) to `false` to drop a column in a table that has data.
```

Comment:
This can be a direct link to the session variable e.g. https://www.cockroachlabs.com/docs/stable/set-vars.html#sql-safe-updates